core/lib/zksync_core/src/api_conversion_rate/mod.rs

```diff
@@ -0,0 +1,38 @@
+use axum::{extract, extract::Json, routing::get, Router};
+use rand::Rng;
+use tokio::sync::watch;
+use zksync_config::configs::native_token_fetcher::NativeTokenFetcherConfig;
+
+pub(crate) async fn run_server(
+    mut stop_receiver: watch::Receiver<bool>,
+    server_configs: &NativeTokenFetcherConfig,
+) -> anyhow::Result<()> {
+    let app = Router::new().route("/conversion_rate/:token_address", get(get_conversion_rate));
+
+    let bind_address = if server_configs.host.starts_with("http://") {
+        &server_configs.host[7..] // If it starts with "http://", strip the prefix
+    } else {
+        &server_configs.host // Otherwise, return the original string
+    };
+
+    axum::Server::bind(&bind_address.parse().expect("Unable to parse socket address"))
+        .serve(app.into_make_service())
+        .with_graceful_shutdown(async move {
+            if stop_receiver.changed().await.is_err() {
+                tracing::warn!("Stop signal sender for conversion rate API server was dropped without sending a signal");
+            }
+            tracing::info!("Stop signal received, conversion rate server is shutting down");
+        })
+        .await
+        .expect("Conversion rate server failed");
+    tracing::info!("Conversion rate server shut down");
+    Ok(())
+}
+
+// basic handler that responds with a static string
+async fn get_conversion_rate(extract::Path(_token_address): extract::Path<String>) -> Json<u64> {
+    tracing::info!("Received request for conversion rate");
+    let mut rng = rand::thread_rng();
+    let random_number: u64 = rng.gen_range(1..=100);
```

Comment:
I think it'll be too volatile for any actual usage. Let's hardcode this as a const for now and later alter the logic as soon as it's needed.

Reply:
Addressed in latest commit:
```rs
async fn get_conversion_rate(extract::Path(_token_address): extract::Path<String>) -> Json<u64> {
    tracing::info!("Received request for conversion rate");
    Json(42)
}
```